### PR TITLE
direct links to files shared on "Public" folder

### PR DESCRIPTION
**Pull request by EazyServer**
Could you please accept this PR. Adding support to direct links on "Public" Folder
Thanks

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Dropbox already support doing this. My intention is only to boot their API package, and nothing else.

File: src/DropboxManager.php [hunk 1]
```diff
@@ -109,4 +109,40 @@ public function getFactory()
...
+     * @param string $path
+     * @param mixed $userAccountNumber
+     *
+     * @return string
+     */
+    public function getDirectLink($path=null, $userAccountNumber=null)
+    {
+        if(!empty($path))
+        {
+            $publicFolderMeta = self::getMetadata('/Public');
```
Comment (GrahamCampbell) [inline src/DropboxManager.php]: This doesn't work. It uses the wrong connection.

File: src/DropboxManager.php [hunk 2]
```diff
@@ -109,4 +109,40 @@ public function getFactory()
...
+            
+            if(!empty($publicFolderMeta))
+            {
+                if(empty($userAccountNumber))
+                {
+                    $userAccount = self::getAccountInfo();
```
Comment (GrahamCampbell) [inline src/DropboxManager.php]: etc

**Comment by EazyServer** [comment]
I don't think so! the only support is a temp link which expires in few hours

**Comment by GrahamCampbell** [comment]
Well, the manager is the wrong place to add this code I'm afraid.

**Comment by EazyServer** [comment]
would you be kind enough and direct me to the best place to add this?

**Pull request closed without merging**